userapi/storage/postgres/deltas/2022060815233800_userprofile_servername.go

```diff
@@ -0,0 +1,36 @@
+package deltas
+
+import (
+	"database/sql"
+	"fmt"
+
+	"github.com/matrix-org/dendrite/internal/sqlutil"
+	"github.com/matrix-org/gomatrixserverlib"
+)
+
+var serverName gomatrixserverlib.ServerName
+
+func LoadProfilePrimaryKey(m *sqlutil.Migrations, s gomatrixserverlib.ServerName) {
+	serverName = s
+	m.AddMigration(UpProfilePrimaryKey, DownProfilePrimaryKey)
+}
+
+func UpProfilePrimaryKey(tx *sql.Tx) error {
+	_, err := tx.Exec(fmt.Sprintf(`ALTER TABLE account_profiles ADD COLUMN IF NOT EXISTS servername TEXT NOT NULL DEFAULT '%s';
```

Comment:
This is going to sound nitpicky but, in the interests of consistency across the project, can we make this `server_name` instead?